tests/sdk/queries/test_query_filter.py

```diff
@@ -414,3 +419,112 @@ def test_filter_group_when_changed_filter_clause_has_correct_json_representation
         '{"operator":"IS", "term":"term", "value":"value2"},'
         '{"operator":"IS", "term":"term", "value":"value3"}]}'
     )
+
+
+class TestQueryFilterTimestampField:
```

Comment:
There were no tests for class `QueryFilterTimestampField` so had to add new ones.